kubernetes.go

```diff
@@ -88,6 +88,9 @@ type KubernetesClusterUpdateRequest struct {
 	MaintenancePolicy *KubernetesMaintenancePolicy `json:"maintenance_policy,omitempty"`
 	AutoUpgrade       *bool                        `json:"auto_upgrade,omitempty"`
 	SurgeUpgrade      bool                         `json:"surge_upgrade,omitempty"`
+
+	// Convert cluster to run highly available control plane
+	HA bool `json:"ha,omitempty"`
```

Comment:
The type should be `*bool` I think so that users can safely omit it without inadvertently asking for HA disablement (which is not going to be supported for now and thus would lead to an error).

Reply:
+1